docs/server/installation.rst

```diff
@@ -312,19 +312,6 @@ the database and populate it with initial data. This is done by executing the
   (env) $ pootle initdb
 
 
-.. _installation#admin_user:
-
-Creating an admin user
-----------------------
```

Comment:
This is required.

Allauth is setup to require an email.  When you first log in as admin/admin it will claim to be sending you an email but can't because there is no admin user.

Forcing the creation of a superuser ensures that the email is created.